[2021-02-04 14:34:23,281] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: udac_example_dag.Stage_events 2019-01-12T00:00:00+00:00 [queued]>
[2021-02-04 14:34:23,287] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: udac_example_dag.Stage_events 2019-01-12T00:00:00+00:00 [queued]>
[2021-02-04 14:34:23,287] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-02-04 14:34:23,287] {taskinstance.py:881} INFO - Starting attempt 7 of 7
[2021-02-04 14:34:23,287] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-02-04 14:34:23,290] {taskinstance.py:901} INFO - Executing <Task(StageToRedshiftOperator): Stage_events> on 2019-01-12T00:00:00+00:00
[2021-02-04 14:34:23,295] {standard_task_runner.py:54} INFO - Started process 62903 to run task
[2021-02-04 14:34:23,326] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'udac_example_dag', 'Stage_events', '2019-01-12T00:00:00+00:00', '--job_id', '200', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/udac_example_dag.py', '--cfg_path', '/var/folders/ys/hb038j3904j77l4dkb4zk2ym0000gq/T/tmprzs7iiv_']
[2021-02-04 14:34:23,327] {standard_task_runner.py:78} INFO - Job 200: Subtask Stage_events
[2021-02-04 14:34:23,346] {logging_mixin.py:112} INFO - Running <TaskInstance: udac_example_dag.Stage_events 2019-01-12T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-02-04 14:34:23,368] {stage_redshift.py:56} INFO - StageToRedshiftOperator not implemented yet
[2021-02-04 14:34:23,432] {stage_redshift.py:61} INFO - Clearing data from destination Redshift table: staging_events
[2021-02-04 14:34:23,434] {base_hook.py:89} INFO - Using connection to: id: redshift. Host: redshift-cluster-airflow.ca3iipyjqhur.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: None
[2021-02-04 14:34:23,463] {taskinstance.py:1150} ERROR - could not translate host name "redshift-cluster-airflow.ca3iipyjqhur.us-west-2.redshift.amazonaws.com" to address: nodename nor servname provided, or not known
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/trevorransom/Documents/01 Coding/03 Data Engineering/Udacity/05 Airflow/plugins/operators/stage_redshift.py", line 62, in execute
    redshift.run('DELETE FROM {}'.format(self.table))
  File "/usr/local/lib/python3.9/site-packages/airflow/hooks/dbapi_hook.py", line 164, in run
    with closing(self.get_conn()) as conn:
  File "/usr/local/lib/python3.9/site-packages/airflow/hooks/postgres_hook.py", line 93, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/usr/local/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "redshift-cluster-airflow.ca3iipyjqhur.us-west-2.redshift.amazonaws.com" to address: nodename nor servname provided, or not known

[2021-02-04 14:34:23,467] {taskinstance.py:1187} INFO - Marking task as FAILED. dag_id=udac_example_dag, task_id=Stage_events, execution_date=20190112T000000, start_date=20210204T123423, end_date=20210204T123423
[2021-02-04 14:34:28,290] {local_task_job.py:102} INFO - Task exited with return code 1
